crates/bevy_ecs/src/lib.rs

```diff
@@ -152,9 +155,8 @@ mod tests {
     #[derive(Component, Debug, PartialEq, Eq, Clone, Copy)]
     struct C;
 
-    #[expect(dead_code)]
     #[derive(Default)]
-    struct NonSendA(usize, PhantomData<*mut ()>);
+    struct NonSendA(PhantomData<*mut ()>);
```

Comment:
In case you're wondering: I checked to see if this `usize` had any purpose. It did in the past, [when this struct was first added](https://github.com/bevyengine/bevy/commit/0bf7f3153d94543078ef518b2a699b1f24b08dd5#diff-696a3460ea2e00a5e92135018cb36f5bc0e08214a1e24aa0dac608f3c6d3a865R1269-R1278). However, it seems that test has since been removed; for what reason, I'm unsure.

Regardless, the `usize` is no longer needed, so I've removed it.